app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def latest_posts
+    Post.where(author: self).limit(3)
+  end
```

Comment:
- [ ] Good job so far 👏 👏 Your tests look good. In addition to what you have done, I would suggest adding tests for the following custom method in the `User` model that helps you query the latest post. This is among the project requirements as you can see below 👇 
![image](https://user-images.githubusercontent.com/90222110/215150348-f0480b20-1691-49f6-9221-0ec92639b89a.png)